packages/wasm/src/services/credential/bbs-revocation.ts

```diff
@@ -121,6 +59,18 @@ export const getWitnessDetails = async (credential, _membershipWitness) => {
 
   const membershipWitness = new VBMembershipWitness(hexToU8a(witness));
 
+  try {
+    await blockchainService.modules.accumulator.updateWitness(
+      registryId,
+      encodedRevId,
+      membershipWitness,
+      queriedAccumulator.created,
```

Comment:
Shouldn't this use `blockNo` instead of `queriedAccumulator.created`? This might work for credentials that are issued with a brand new accumulator, but if it is using an existing accumulator that had some updates done to it the updates from `created` until `blockNo` shouldn't be applied to this.